### ISSUE TITLE
[3.5 - 3.6] Gradual support in Air at flat surfaces

### ISSUE THREAD

**Issue by TimoErdmann** [Ultimaker/Cura#4973]
**Application Version**
3.6

**Platform**
Windows 10 - 64bit

**Printer**
custom FFM printer
[CFFFP_TestSupport.3mf.zip](https://github.com/Ultimaker/Cura/files/2668096/CFFFP_TestSupport.3mf.zip)

**Steps to Reproduce**
Option1: import FFM 
Option2: import an STL with flat bottom surface which needs support
**Actual Results**
![gradualsupportinair](https://user-images.githubusercontent.com/40594118/49809380-086ece00-fd5f-11e8-9e57-cbc5d1f56693.PNG)
The generated **gradual** support hangs in air. It will wall down and built up. But there is a possibillity that the extruder grabs these strings and fuses them with the part.

**Expected results**
![gradualsupportangular](https://user-images.githubusercontent.com/40594118/49809476-4b30a600-fd5f-11e8-9af2-61eac7340d81.PNG)
expand the gradual support with a defined angle so it builds up to the layer underneath.

**Additional Information**
![img_1268](https://user-images.githubusercontent.com/40594118/49809646-bed2b300-fd5f-11e8-9ace-47bf3759e9ec.jpg)

**Comment by JohnEdwa** [comment]
Another possible implementation that would work fairly well with almost no additional material cost would be using bridging. Cura even has the necessary functionality already - simply generate a new 'support first layer grid' structure between the gradual infill steps using the new support density.

![capture](https://user-images.githubusercontent.com/5417183/49904685-a2c03600-fe73-11e8-89b2-b3085a6b77a3.PNG)

**Comment by TimoErdmann** [comment]
Yes you could use bridging. But not all Materials are cabable of this feature and this then this will fail tremendously. So there can be an option with bridging but there has to be on option for the angled version too.
